Composer/packages/adaptive-form/src/components/FormTitle.tsx

```diff
@@ -164,6 +178,9 @@ const FormTitle: React.FC<FormTitleProps> = (props) => {
             </React.Fragment>
           )}
         </p>
+        <div css={styles.comment}>
+          <Comment key={formData.$designer.id} comment={formData?.$designer?.comment} onChange={handleCommentChange} />
```

Comment:
key can be undefined?

Reply:
Yes. This in reality shouldn't be because we generate the id property for each action. This is here to force the component to update though. The form has some optimizations which makes this necessary.